ui/components/src/GaugeChart/GaugeChart.tsx

```diff
@@ -112,7 +120,11 @@ export function GaugeChart(props: GaugeChartProps) {
           min: 0,
           max,
           pointer: {
-            show: false,
+            show: true,
+            icon: 'path://M12.8,0.7l12,40.1H0.7L12.8,0.7z',
```

Comment:
Is this the little triangle? How did the path value get generated for this? Mostly curious for future us if we ever want to adjust it.